staging/src/k8s.io/client-go/tools/auth/exec/exec_test.go

```diff
@@ -139,6 +233,11 @@ func TestLoadExecCredential(t *testing.T) {
 			}),
 			wantErrorPrefix: "cannot create rest.Config",
 		},
+		{
+			name:            "v1 nil cluster",
+			data:            marshal(t, clientauthenticationv1.SchemeGroupVersion, &clientauthenticationv1.ExecCredential{}),
+			wantErrorPrefix: "ExecCredential does not contain cluster information",
+		},
```

Comment:
?

https://github.com/kubernetes/kubernetes/blob/1345a802dee9c0b0b952d5ec488ea224ebe0b804/staging/src/k8s.io/client-go/tools/auth/exec/exec.go#L70-L71

Reply:
tried to remember if there was some PR convo about this, but i don't remember one. i could have sworn i talked to jordan about it at some point. here is where it came in: https://github.com/kubernetes/kubernetes/pull/95489/files#diff-51669535eb7c209fdb6dc6e2db51fd89e52a999f7c31244af8dd3558181ea4d5R71

tried to fix it here: 6879a94624b3ab605658d4a5b2ab125e8ba43f93